spring-cloud-task-samples/batch-events/pom.xml

```diff
@@ -50,11 +51,13 @@
 		<dependency>
 			<groupId>org.springframework.cloud</groupId>
 			<artifactId>spring-cloud-starter-stream-rabbit</artifactId>
+			<version>1.0.2.RELEASE</version>
```

Comment:
With the addition of adding specific version numbers to our samples, it makes me wonder if we should move samples into their own repo.